### PR TITLE
fix: use regional terrain for `gym_fitness_1`

### PR DESCRIPTION
**Pull request by 0Monet**
### Required

- [X] I wrote the PR title in [conventional commit format](https://docs.cataclysmbn.org/en/contribute/changelog_guidelines/).
- [X] I ran the [code formatter](https://docs.cataclysmbn.org/en/contribute/contributing/#code-style).
- [X] I linked any relevant issues using [github keyword syntax](https://docs.cataclysmbn.org/en/contribute/contributing/#pull-request-notes) so it can be closed automatically.
- [X] I have [committed my changes to new branch that isn't `main`](https://docs.cataclysmbn.org/en/contribute/contributing/#make-your-changes) so it won't cause conflict when updating `main` branch later.

## Purpose of change
Use regional terrain for the mapgen `gym_fitness_1`
## Describe the solution
Replace non-regional terrain with regional pseudo terrain.
## Describe alternatives you've considered
none
## Additional context
Before:
image1
After:
image2

### CODE REVIEW

File: data/json/mapgen/gym.json [hunk 1]
```diff
@@ -132,40 +132,40 @@
...
-        ".|{ V V  |::::|||||||||.",
-        ".|{      |:~~::::::::|4.",
-        ".|{ V V  5:~~~~~~~~~:|..",
-        ".|{      =:~~~~~~~~~:|..",
-        ".|{ V V  5:~~~~~~~~~:5..",
-        ".|||||||||:~~~~~~~~~:5..",
-        ".........|:~~~~~~~~~:|..",
-        ".........|:::::::::::|..",
-        ".........|||5555555|||.."
+        "              |--===--|'",
```
Comment (scarf005) [inline data/json/mapgen/gym.json:135]:
```suggestion
        "              |--===--| ",
```

File: data/json/mapgen/gym.json [hunk 2]
```diff
@@ -132,40 +132,40 @@
...
+        ".": "t_floor",
         "~": "t_water_pool",
         ":": "t_linoleum_white",
         "*": "t_linoleum_white",
         "+": "t_door_c",
-        ".": [ "t_grass", "t_grass", "t_grass", "t_grass", "t_grass", "t_shrub", "t_dirt" ],
-        "'": "t_dirt",
-        "5": "t_wall_glass",
+        " ": "t_region_groundcover_urban",
+        "'": "t_region_groundcover_urban",
```
Comment (scarf005) [inline data/json/mapgen/gym.json:167]:
```suggestion
        " ": "t_region_groundcover_urban",
```

**Review by scarf005 — COMMENTED** [review]
`'` and ` ` both seems to do the same, maybe it could be simplified?

File: data/json/mapgen/gym.json (hunk 1 above)
Comment (0Monet) [reply data/json/mapgen/gym.json:135]: They don't do the same thing because `'` is also a street light.

Comment (0Monet) [reply data/json/mapgen/gym.json:135]: image3

**Review by scarf005 — APPROVED** [review]

**Pull request merged**